docs/Benodigde-tabellen.md

```diff
@@ -1,21 +1,49 @@
+| Geslacht                     | LO GBA element 04.10 | Geeft aan wat het geslacht is van de persoon.                                                                                                              |
+| Gezagsverhouding             | Landelijke tabel 61  | Gezagsverhouding                                                                                                                                           |
+| Landen                       | Landelijke tabel 34  | Landen                                                                                                                                                     |
+| Naamgebruik                  | LO GBA element 61.10 | De manier waarop de geslachtsnaam van persoon en partner van persoon moet worden verwerkt in de manier waarop persoon wil worden aangesproken.             |
+| Nationaliteiten              | Landelijke tabel 32  | Nationaliteiten                                                                                                                                            |
+| Reden_Nationaliteit          | Landelijke tabel 37  | Reden opnemen - beeindigen Nationaliteit                                                                                                                   |
+| Reden_Opschorting_Bijhouding | LO GBA element 67.20 | Reden voor opschorting van de bijhouding.                                                                                                                  |
+| Soort_Verbintenis            | LO GBA element 15.10 | Soort verbintenis tussen de persoon en de partner die bij de burgerlijke stand is ingeschreven.                                                            |
+| Verblijfstitel               | Landelijke tabel 56  | Verblijfstitel                                                                                                                                             |
+
+## BRP-historie
+
+Voor de BRP-historie API moeten minimaal de volgende tabellen beschikbaar worden gemaakt:
+
+| tabelidentificatie        | bron                | omschrijving                                                           |
+|---------------------------|---------------------|------------------------------------------------------------------------|
+| Reden_Beeindigen_Huwelijk | Landelijke tabel 41 | Reden ontbinding nietigverklaring huwelijk geregistreerd partnerschap. |
+
+## BRP-Bewoning
+
+Voor de BRP-Bewoning API moeten minimaal de volgende tabellen beschikbaar worden gemaakt:
+| tabelidentificatie      | bron             | omschrijving                                            |
+|-------------------------|------------------|---------------------------------------------------------|
+| Reden_Bewoning_Onbekend | tabelwaarden.csv | Reden dat bewoning en bewoners niet kan worden bepaald. |
```

Comment:
Ook de tabel 'Land' is nodig.

Reply:
zie vorige opmerking

Reply:
Prima, al heeft BRP Bewoning niet alle tabellen nodig die BRP Personen Bevragen nodig heeft.